src/transaction/log_page_buffer.c

```diff
@@ -2482,6 +2482,13 @@ logpb_fetch_start_append_page (THREAD_ENTRY * thread_p)
 
   logpb_log ("started logpb_fetch_start_append_page\n");
 
+  if (prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) != 0
+      && (LOG_PAGEID) prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) == log_Gl.hdr.append_lsa.pageid
```

Comment:
Then, it can also be combined with the if-condition below like:

`f ((log_Gl.hdr.append_lsa.pageid == prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID)) && (log_Gl.hdr.append_lsa.offset == 0))`

Reply:
Ah, joohok already pointed it out.